modules/ROOT/pages/dataweave-cookbook-merge-multiple-payloads.adoc

```diff
@@ -98,61 +38,79 @@ The whole point of this example is that the input is in separate objects.
     "author":"jane doe"
   }
 ]
+output application/json
+---
+firstInput map (firstInputValue) ->
+  {
+    theId : firstInputValue.bookId as Number,
+    theTitle: firstInputValue.title,
+    thePrice: firstInputValue.price as Number,
+    (secondInput filter ($.*bookId contains firstInputValue.bookId) map (secondInputValue) -> {
+      theAuthor : secondInputValue.author
+    })
+  }
 ----
 
+As the next script shows, you can also write the same script using an `id` alias (created with `using (id = firstInputValue.bookId)`). The alias replaces the  selector expression, `firstInputValue.bookId`, which is longer.
```

Comment:
As the next script shows, you can --> The following script shows how you can write....